Extension/SourceEditorCommand.swift

```diff
@@ -4,24 +4,21 @@ import XcodeKit
 class SourceEditorCommand: NSObject, XCSourceEditorCommand {
 
     func perform(with invocation: XCSourceEditorCommandInvocation,
-                 completionHandler: @escaping (Error?) -> Swift.Void) {
+        completionHandler: @escaping (Error?) -> Swift.Void) {
 
         let uti = invocation.buffer.contentUTI
-        if uti != "com.apple.dt.playground" && uti != "public.swift-source" && uti != "com.apple.dt.playgroundpage"{
+        guard uti == "com.apple.dt.playground" || uti == "public.swift-source" || uti == "com.apple.dt.playgroundpage" else {
```

Comment:
There is no bug now definitely.
What I mean is that multi `guard let` is too complicate to understand.
```swift
guard let utiList.contain(uti) else {
    return
}
```